### PR TITLE
adding templates for bug reports, feature requests, and pull requests

### PR DESCRIPTION
**Pull request by dsavransky**
Adding standardized templates for bug reports, feature requests, and pull requests.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+## Type of change
+
+Please delete options that are not relevant (and this line).
+
+- Bug fix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- This change requires a documentation update
+
+## Reference any relevant issues (don't forget the #)
```
Comment (semaphoreP) [inline .github/pull_request_template.md:13]: do we need this? I think you can reference issues as part of the "Development" sidebar (although doing so does close the issue upon merge of PR).

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+
+## Type of change
+
+Please delete options that are not relevant (and this line).
+
+- Bug fix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- This change requires a documentation update
```
Comment (semaphoreP) [inline .github/pull_request_template.md:11]: I don't understand this particular kind of change. shouldn't the documentation update be part of the PR

**Review by semaphoreP — COMMENTED** [review]
Had a couple of minor questions on the PR one. otherwise, looks good. Thanks for putting this together.

File: .github/pull_request_template.md (hunk 1 above)
Comment (dsavransky) [reply .github/pull_request_template.md:13]: I think I got this from a default pull request template a while ago.  It means that the pull pops up in the history of an issue even before its accepted/merged, possibly avoiding duplicative effort.  We can remove it, if you prefer.

File: .github/pull_request_template.md (hunk 2 above)
Comment (dsavransky) [reply .github/pull_request_template.md:11]: I'll remove that option.

File: .github/pull_request_template.md (hunk 1 above)
Comment (semaphoreP) [reply .github/pull_request_template.md:12]: Let's try it for now, and if it feels too redundant, we can remove.

**Review by semaphoreP — APPROVED** [review]

**Pull request merged**